docs/online/src/scenarios/postmessage.rst

```diff
@@ -269,6 +270,76 @@ You can send the following messages; all others are ignored:
         }
 
 
+..  data:: Host_RefreshSessionInfo
+
+    When the host wants to send new tokens to |wac|, for instance when an :term:`access token` is about to expire, 
+    a :data:`Host_RefreshSessionInfo` message is posted. The host can send a set of new access tokens and their properties 
+    in this post message. 
+
+    ..  note:: 
+        |wac| expects a :data:`Host_RefreshSessionInfo` response to a :data:`App_RefreshSessionInfo` request. If no response is 
+        received by |wac|, the session could expire. 
+    
+    If the host is unable to issue new tokens an error message must be provided, with an optional description of the 
+    reason. If no errors are present, the value property must contain the token and its expiry. 
+
+    ..  note::
+        The host must indicate the support for :data:`App_RefreshSessionInfo` by setting the :term:`SupportsSessionRefresh` parameter 
+        in the :ref:`CheckFileInfo` response to true.
+
+    .. attribute:: Values
+        :noindex:
+
+        Tokens  *(JSON-formatted list)*
+            :term:`Tokens` refers to an array of *(JSON-formatted object)* identifying new :term:`access token` sent to |wac|.
+
+            Name *(string)*
+            :term:`Name` refers to the property used to identify token elements.
+
+            Expiry *(long)*
+            :term:`Expiry` refers to the time the token would no longer be valid.
+
+            Value *(string)*
+            :term: `Value` refers to the value of the new token.
+
+            Errors *(JSON-formated list)*
+
+                Id *(string)*
+                :term:`Id` refers to a *(string)* indicating error type.
+
+                Details *(string)*
+                :term:`Details` refers to any additional details the host wishes to provide for debugging purposes.
+
+    ..  rubric:: Example Message:
+    ..  .. important:: |wac| supports the :term:`access token` of type "AccessToken" (Word, Powerpoint and Excel) and "FileGetUrl" (Powerpoint)
```

Comment:
If these are the only supported values then they should be specced as such. See here for an example: https://wopi.readthedocs.io/projects/wopirest/en/latest/files/CheckFileInfo.html#term-supportedshareurltypes